scripts/utils.sh

```diff
@@ -224,6 +224,18 @@ get_hopr_address() {
   try_cmd "${cmd}" 30 5 | jq -r ".hopr"
 }
 
+# $1 = optional: endpoint, defaults to http://localhost:3001
+get_multiaddress(){
```

Comment:
Not being used anywhere?

Reply:
true, I used that back when we needed to use mutliaddresses in the network registry, should be keep it in but comment it out?

Reply:
I'd rather remove it entirely in that case.

Reply:
removed, please check & resolve